package.json

```diff
@@ -18,15 +18,20 @@
     "build": "rm -rf lib mod && tsc -p tsconfig.build.json && tsc -p tsconfig.module.json",
     "lint": "tslint -c tslint.json -p tsconfig.json",
     "test": "npm run lint && jest",
-    "prepublishOnly": "npm run test && npm run build"
+    "prepublishOnly": "npm run test && npm run build",
+    "benchmark": "ts-node bench.ts"
   },
   "devDependencies": {
+    "@types/benchmark": "^1.0.31",
     "@types/jest": "^23.3.2",
     "@types/node": "^10.12.0",
     "@types/pngjs": "^3.3.0",
+    "benchmark": "^2.1.4",
     "jest": "^23.6.0",
+    "microtime": "^2.1.8",
```

Comment:
accidental leftover dependency from a previous commit?

Reply:
Looks like it's an optional dependency of benchmarkJS.

Reply:
That's correct. I added it to get better benchmark precision.